apps/server/src/schema/feedback/feedback-types.ts

```diff
@@ -0,0 +1,18 @@
+import { builder } from '../builder';
+
+export const FeedbackTypeEnum = builder.enumType('FeedbackType', {
+  values: {
+    BUG_REPORT: { description: 'Report a bug' },
+    FEATURE_SUGGESTION: { description: 'Suggest a feature' },
+    OTHER: { description: 'Other feedback' },
+  },
+});
```

Comment:
```suggestion
export const FeedbackTypeEnum = builder.enumType(FeedbackType, {
  name: 'FeedbackTypeEnum',
});
```
We don't need cat is cat descriptions https://www.reddit.com/r/ProgrammerHumor/comments/8w54mx/code_comments_be_like/